global.json

```diff
@@ -0,0 +1,6 @@
+{
+  "sdk": {
```

Comment:
Not sure we want this? @smoogipoo

Reply:
Theoretically, there are environments that can compile `net6.0` but have unfixed issues in the SDK.
If we introduce `net7.0` in any project, I think it can serve as a soft requirement for SDK version.

Reply:
@smoogipoo can you make a call on this one? from our brief discussion last week i think you'd want to see this removed?

Reply:
I think I would prefer this being removed, since it's one of those things that silently changes compilation in the background that isn't easily noticeable via any tooling.